RecoLocalCalo/HcalRecAlgos/interface/MahiFit.h

```diff
@@ -0,0 +1,165 @@
+#ifndef RecoLocalCalo_HcalRecAlgos_MahiFit_HH
+#define RecoLocalCalo_HcalRecAlgos_MahiFit_HH
+
+#include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+#include "CalibFormats/HcalObjects/interface/HcalCalibrations.h"
+#include "CalibFormats/HcalObjects/interface/HcalCoder.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/EigenMatrixTypes.h"
+#include "DataFormats/HcalRecHit/interface/HBHEChannelInfo.h"
+
+#include "CalibCalorimetry/HcalAlgos/interface/HcalPulseShapes.h"
+#include "CalibCalorimetry/HcalAlgos/interface/HcalTimeSlew.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFunctor.h"
+
+#include <Math/Functor.h>
+
+struct nnlsWorkspace {
```

Comment:
Please capitalize [NNLSWorkspace or NnlsWorkspace]

Reply:
the definition better be moved inside MahiFit
or named more specifically, e.g. MahiNnlsWorkspace